src/containers/shared/workflow-status-tag/workflow-status-tag-icon/workflow-status-tag-icon.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { withStyle } from 'baseui';
+import { Spinner } from 'baseui/spinner';
+
+export const styled = {
```

Comment:
This name doesn't work if we import "styled" from baseui.
Any suggestions for alternatives?

Reply:
why not alias baseweb styled ?

Reply:
Yup that's what I did later, we discussed this offline 

I'll close this thread